samples/dashboards/demo/live-weather/demo.html

```diff
@@ -0,0 +1,44 @@
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/maps/modules/map.js"></script>
+<script src="https://code.highcharts.com/modules/solid-gauge.js"></script>
+<script src="https://code.highcharts.com/modules/timeline.js"></script>
```

Comment:
Do we need this series?

Reply:
This series contains the geographical information (index 1 on the map) so I can't see how it can be removed. I could however do away with the JSON format and put the info directly in the data and thus save some code.

Reply:
I mean that the reference to the `timeline.js` module is unnecessary :)

Reply:
> I mean that the reference to the `timeline.js` module is unnecessary :)

That makes sense! I thought it would be extremely heavy-handed to remove the entire series :-)